src/clang-c-frontend/clang_c_convert.h

```diff
@@ -91,7 +93,14 @@ class clang_c_convertert
 
   virtual bool get_function(const clang::FunctionDecl &fd, exprt &new_expr);
 
-  bool get_function_params(const clang::ParmVarDecl &pd, exprt &param);
+  virtual bool
+  get_function_body(const clang::FunctionDecl &fd, exprt &new_expr);
```

Comment:
Shall we document these new methods?

Reply:
Done. 
Explained why they are made virtual.